reference/forms/types/range.rst

```diff
@@ -0,0 +1,61 @@
+.. index::
+   single: Forms; Fields; range
+
+range Field Type
+================
+
+The ``range`` field is a slider that is rendered using the HTML5
+``<input type="range" />`` tag.
+
++-------------+---------------------------------------------------------------------+
+| Rendered as | ``input`` ``range`` field (slider in HTML5 supported browser)       |
++-------------+---------------------------------------------------------------------+
+| Inherited   | - `data`_                                                           |
+| options     | - `disabled`_                                                       |
+|             | - `empty_data`_                                                     |
```

Comment:
`EmailType` and `RangeType` both have the same type as a parent; **text** (`TextType`).

The parent type of **text** is **form** (`FormType`). The inherited options are from this **form** type:

http://symfony.com/doc/current/reference/forms/types/text.html